src/apps/btc/btc_common.h

```diff
@@ -131,6 +131,7 @@ USE_RESULT bool btc_common_address_from_payload(
  * @param[inout] pk_script_len: size of pk_script IN, size of the resulting pk_script OUT.
  */
 USE_RESULT bool btc_common_pkscript_from_payload(
+    const app_btc_coin_params_t* params,
     BTCOutputType output_type,
     const uint8_t* payload,
```

Comment:
the doc comment says "payload pubkey hash or script hash" but isn't it just a pubkey for P2TR?

Reply:
yes, true! fixed